compress_string.cpp

```diff
@@ -32,7 +32,7 @@ std::string compress(const std::string &to_compress)
 			ss << c;
 		}
 	}
-
+	ss << count; // add the count for the last char
```

Comment:
nicely spotted. Testing one's code is necessary! Have you looked at Catch yet? It's awesome. https://github.com/philsquared/Catch